scionlab/tests/test_config_tar.py

```diff
@@ -63,6 +63,13 @@ def test_user_as(self):
             self._check_archive('user_as_%i' % user_as.id, archive)
 
     def _check_archive(self, test_id, archive):
+
+        unchecked_files = ["README.md"]
+        # Ignore content for `unchecked_files`
+        for fname in unchecked_files:
+            if fname in archive.dict:
+                archive.dict[fname] = "content_not_checked"
```

Comment:
DictWriter writes to a flat dict, so if a file is ignored in one level of the archive, it is only represented once and hence ignored?

Reply:
No, this will only ignore the content of `README.md` at the root of the archive not in sub-directories (if that was the question). The keys in the dict are the full path in the archive.